### PR TITLE
HomePage Styling & Date Control

### PR DESCRIPTION
**Pull request by JakeAdmire**
## Changes Made:
- Added actions for `loading` & `semanticDate`
- Added functionality within `Calendar.js` for determining semantic date
- Added `semanticDateReducer.js` for controlling `semanticDate` state
- Fixed naming convention within `dateReducer.js`
- Added `loading.gif` image and implemented into `HomePage`
- Styled changes

### Relevant Screenshots:
N/A

### Message for reviewer:
Please review these changes

### Testing complete?
- [ ] Yes
- [x] Not yet
- [ ] N/A

### Final checklist:
- [x] My code has no unused/commented out code
- [x] I have reviewed my code
- [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/Components/Container/Container.js [hunk 1]
```diff
@@ -19,50 +20,111 @@ const theme = {
...
     const { date, routines, loading } = this.props;
 
     return loading
-      ? <Text>loading..</Text>
-      : <View>
-        {this.displayCards()}
-      </View>
+
+      ? <View style={styles.loadingContainer}>
+          <Image style={styles.loader} source={require('../../../assets/images/loading.gif')} />
```
Comment (DevelopingDavid) [inline src/Components/Container/Container.js:71]: love the gif idea

File: src/Components/Container/Container.js [hunk 2]
```diff
@@ -19,50 +20,111 @@ const theme = {
...
   date: state.date,
+  semanticDate: state.semanticDate,
   routines: state.routines,
   loading: state.loading
 })
 
 export default connect(mapStateToProps)(Container);
+
+const styles = StyleSheet.create({
+
```
Comment (DevelopingDavid) [inline src/Components/Container/Container.js:88]: good naming conventions

File: src/Components/Calendar/Calendar.js [hunk 3]
```diff
@@ -1,67 +1,126 @@
...
+import { RalewayText, RalewayBoldText } from '../../Utilities/RalewayText';
+import { setDate, loadRoutines, setSemanticDate } from '../../Actions/index'
 import { fetchRoutines } from '../../Thunks/fetchRoutines';
-import Container from '../Container/Container';
+
+
 
 export class Calendar extends Component {
 
+  componentDidMount = () => {
```
Comment (ericfitzsimons451) [inline src/Components/Calendar/Calendar.js:16]: Way to add these to the calendar page.  Looks really cool.

File: src/Components/Calendar/Calendar.js [hunk 4]
```diff
@@ -1,67 +1,126 @@
...
+  determineDay = (day) => {
+    let dayNum = day.split('')[0] === '0' ? day.slice(1, 2) : day;
+    let letters = [ 'th', 'st', 'nd', 'rd', 'th', 'th', 'th', 'th', 'th', 'th' ];
+    let lastDigit = day.split('').reverse()[0];
+    return `${dayNum}${letters[lastDigit]}`;
+  }
+
+  determineMonth = (month) => {
+    let monthNum = month.split('')[0] === '0' ? month.slice(1, 2) : month;
+    let months = [ 'January', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December'];
```
Comment (ericfitzsimons451) [inline src/Components/Calendar/Calendar.js:39]: Right now, when May shows up, it appears as Mav cause it gets cut off.  We should fix this with a bit more padding, or just enlarge the box.

File: src/Components/Container/Container.js (hunk 1 above)
Comment (ericfitzsimons451) [inline src/Components/Container/Container.js:71]: Loading gif!  Hero move!

**Comment by ericfitzsimons451** [comment]
Nice work, Jake!  We're in discussions with Tim right now to figure out some endpoint issues regarding request formats and where they will be sent.

**Pull request merged**